components/Snackbar/examples/supplemental/SnackbarExampleSupplemental.h

```diff
@@ -30,6 +30,10 @@
 /** The floating action button shown in the bottom right corner. */
 @property(nonatomic) MDCFloatingButton *floatingButton;
 
```

Comment:
It isn't really necessary here... I just removed the comment from the floatingButton property.